### PR TITLE
[RA-3066] Add FileSizeInBytes to FileManager

### PR DESCRIPTION
**Pull request by gaborpongracz**

### CODE REVIEW

File: fileutil/fileutil.go [hunk 1]
```diff
@@ -89,3 +90,16 @@ func (fileManager) ensureSavePath(savePath string) error {
...
+func (fileManager) FileSizeInBytes(pth string) (float64, error) {
+	if pth == "" {
+		return 0, errors.New("No path provided")
+	}
+	fileInf, err := os.Lstat(pth)
+	if err != nil {
+		return 0, err
+	}
+
+	return float64(fileInf.Size()), nil
```
Comment (godrei) [inline fileutil/fileutil.go:104]: FileInfo.Size is an int64, do we really need it as a float?

File: fileutil/fileutil_test.go [hunk 2]
```diff
@@ -5,8 +5,9 @@ import (
 	"path/filepath"
 	"testing"
 
-	"github.com/bitrise-io/go-utils/v2/pathutil"
 	"github.com/stretchr/testify/require"
+
```
Comment (godrei) [inline fileutil/fileutil_test.go:9]: nit: unnecessary empty line

File: retryhttp/retryhttp.go [hunk 3]
```diff
@@ -1,8 +1,9 @@
 package retryhttp
 
 import (
-	"github.com/bitrise-io/go-utils/v2/log"
 	"github.com/hashicorp/go-retryablehttp"
+
```
Comment (godrei) [inline retryhttp/retryhttp.go:5]: nit: unnecessary empty line

File: fileutil/fileutil.go (hunk 1 above)
Comment (gaborpongracz) [reply fileutil/fileutil.go]: Updated.

File: retryhttp/retryhttp.go (hunk 3 above)
Comment (gaborpongracz) [reply retryhttp/retryhttp.go]: Fixed.

**Pull request merged**